### PR TITLE
EVP Automated Test LPS-186802|master

### PR DESCRIPTION
**Pull request by juliosilveira77**
https://liferay.atlassian.net/browse/LPS-186802

### CODE REVIEW

**Comment by juliosilveira77** [comment]
ci:test:sf

**Comment by juliosilveira77** [comment]
ci:test:sf

**Comment by renato-araujo-liferay** [comment]
ci:test:osb-site-initializer-evp

**Comment by renato-araujo-liferay** [comment]
ci:test:osb-site-initializer-evp

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/EVPOrganization.macro [hunk 1]
```diff
@@ -112,12 +112,13 @@ definition {
...
 		}
-		else {
+
+		if (${done} == "true") {
 			Button.click(button = "Done");
 		}
 	}
 
-	@summary = "Add a new organization"
+	@summary = "Go to create an organization"
```
Comment (renato-araujo-liferay) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/EVPOrganization.macro:121]: Hey, @juliosilveira77 could you please set a better summary?
reading full context this one sounds vague.
Suggestion: "Add new organization in organization page"

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase [hunk 2]
```diff
@@ -21,6 +21,12 @@ definition {
 			EVPUtils.addEVPSite();
```
Comment (renato-araujo-liferay) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase:21]: Hey @juliosilveira77, none of these changes modified the LPS-186802 as mentioned in the commit title.
could you please set a appropriate commit title?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase [hunk 3]
```diff
@@ -39,47 +45,75 @@ definition {
...
+			MenuItem.click(menuItem = "Review");
+
+			EVPOrganization.changeStatus(
+				button = "Request more Info",
+				typeMessage = "Good Morning, Good Afternoon and Good Night");
 		}
 
-		task ("Then the edit button dont appear") {
-			MenuItem.viewNotPresent(menuItem = "Edit");
+		task ("Then The Organization status will change for 'Awaiting More Info From Employee'") {
```
Comment (renato-araujo-liferay) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase:91]: @juliosilveira77 , adjust the task description as above suggestion

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase [hunk 4]
```diff
@@ -39,47 +45,75 @@ definition {
...
+				organizationName = "Organization 5",
+				status = "Awaiting More Info From Employee");
 		}
-	}
 
-	@description = "This is a test for LPS-172949. Make sure the edit button does not appear when the user's role is 'User manager'."
-	@priority = 5
-	test ManagerCannotEditOrganizationInAwatingApprovalStatus {
-		task ("Given the Manager user is in the Organization page") {
+		task ("When the Employee edit an Organization that is with 'Awaiting More Info From Employee' status") {
```
Comment (renato-araujo-liferay) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase:97]: Hey @juliosilveira77 , It's a good idea to inform in the task description of the user change process and his edit journey

**Comment by juliosilveira77** [comment]
ci:test:sf

**Comment by juliosilveira77** [comment]
ci:test:sf

**Comment by juliosilveira77** [comment]
ci:test:sf

**Pull request closed without merging**